playbooks/byo/openshift-checks/adhoc.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: OpenShift health checks
+  hosts: OSEv3
+  roles:
+  - openshift_health_checker
+  vars:
+  - r_openshift_health_checker_playbook_context: adhoc
+  pre_tasks:
+  - name: Print list of known health checks
+    action: openshift_health_check
+    when: checks is undefined or not checks
+  post_tasks:
+  - name: Run health checks
+    action: openshift_health_check
+    args:
+      checks: '{{ checks | default([]) }}'
```

Comment:
What we pass in as an arg to the action plugin is irrelevant, as that's not an inventory variable. But we don't have a `checks` inventory variable before this PR so there's no migration necessary to change that.

You think that command line is long until you type the commands for running via the container... :)

I think the user should have `openshift_checks` for consistency with `openshift_disable_checks`...

    args:
      checks: '{{ openshift_checks | default([]) }}'